dask_planner/src/sql/table.rs

```diff
@@ -188,7 +188,7 @@ pub(crate) fn table_from_logical_plan(
 
             let table_ref: TableReference = table_scan.table_name.as_str().into();
             let (schema, tbl) = match table_ref {
-                TableReference::Bare { table } => ("", table),
+                TableReference::Bare { table } => (Cow::from(""), table),
```

Comment:
Cool, I guess the other choice is to do:
```
TableReference::Bare { table } => ("".to_string(), table),
                TableReference::Partial { schema, table } => (schema.to_string(), table),
                TableReference::Full {
                    catalog: _,
                    schema,
                    table,
                } => (schema.to_string(), table),
```
to avoid the extra import, but I don't really have a strong opinion one way or another.